packages/admin/cms-admin/src/dam/DataGrid/selection/DamMoreActions.tsx

```diff
@@ -205,9 +205,10 @@ const NumberSelectedChip = styled("div")`
     display: flex;
     align-items: center;
     height: 24px;
+    width: 24px;
     background-color: ${({ theme }) => theme.palette.primary.main};
     margin-left: 10px;
-    padding: 0 10px;
+    justify-content: center;
     border-radius: 12px;
```

Comment:
Are these style changes still needed?

Reply:
Yeah the are needed to ensure that the chip is displayed as a circle and that problem appeared in both cases